### DISCUSSION TITLE
Mark some items in a circuit with a different color ?

### DISCUSSION THREAD

**Discussion by jongena** [hneemann/Digital#1007] [Q&A]
Is there a possibility to mark certain items in a circuit with a different color than black ? (red? blue? green... ? )

**Comment by hneemann** [comment]
Only sub circuits can be colored.
However, texts and boxes can be added to describe parts of the circuit.

**Reply by jongena** [reply]
Yes, I have seen that and used this option already. It would however be nice (or even needed sometimes) to be able to use another
color than black for part of the circuit. This e.g. to indicate a group of components that execute one specific task, to highlight that 
parts still need further development,....  etc.
